tensorflow/compiler/xla/service/custom_call_target_registry.cc

```diff
@@ -17,6 +17,13 @@ limitations under the License.
 
 namespace xla {
 
+#if GOOGLE_CUDA
+  extern const std::string kGPUPlatform = "CUDA"; 
```

Comment:
Should that be in custom call registry though ? Maybe we should use a more generic header and use it in more places?

Reply:
I have addressed this by using function from PlatformUtil. Please see the latest commit